system/Database/OCI8/Forge.php

```diff
@@ -273,9 +274,28 @@ protected function _dropTable(string $table, bool $ifExists, bool $cascade)
         return $sql;
     }
 
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table. Also returns array instead of string.